interfaces/console/src/components/NodeNetworkTab/index.tsx

```diff
@@ -87,56 +104,29 @@ const NodeNetworkTab = ({
           }}
         >
           <Stack spacing={4}>
-            <LineChart
-              nodeId={nodeId}
-              loading={loading}
-              metricFrom={metricFrom}
-              topic={NetworkChartsConfig[0].id}
-              title={NetworkChartsConfig[0].name}
-              tabSection={Graphs_Type.Network}
-              initData={getMetricValue(NetworkChartsConfig[0].id, metrics)}
-              hasData={isMetricValue(NetworkChartsConfig[0].id, metrics)}
-            />
-            <LineChart
-              nodeId={nodeId}
-              loading={loading}
-              metricFrom={metricFrom}
-              title={NetworkChartsConfig[1].name}
-              tabSection={Graphs_Type.Network}
-              topic={NetworkChartsConfig[1].id}
-              initData={getMetricValue(NetworkChartsConfig[1].id, metrics)}
-              hasData={isMetricValue(NetworkChartsConfig[1].id, metrics)}
-            />
-            <LineChart
-              nodeId={nodeId}
-              topic={NetworkChartsConfig[2].id}
-              title={NetworkChartsConfig[2].name}
-              loading={loading}
-              metricFrom={metricFrom}
-              tabSection={Graphs_Type.Network}
-              initData={getMetricValue(NetworkChartsConfig[2].id, metrics)}
-              hasData={isMetricValue(NetworkChartsConfig[2].id, metrics)}
-            />
-            <LineChart
-              nodeId={nodeId}
-              topic={NetworkChartsConfig[3].id}
-              title={NetworkChartsConfig[3].name}
-              loading={loading}
-              metricFrom={metricFrom}
-              tabSection={Graphs_Type.Network}
-              initData={getMetricValue(NetworkChartsConfig[3].id, metrics)}
-              hasData={isMetricValue(NetworkChartsConfig[3].id, metrics)}
-            />
-            <LineChart
-              nodeId={nodeId}
-              topic={NetworkChartsConfig[4].id}
-              title={NetworkChartsConfig[4].name}
-              loading={loading}
-              metricFrom={metricFrom}
-              tabSection={Graphs_Type.Network}
-              initData={getMetricValue(NetworkChartsConfig[4].id, metrics)}
-              hasData={isMetricValue(NetworkChartsConfig[4].id, metrics)}
-            />
+            {selected === 1
+              ? networkBackhaul.map((config, i) => (
+                  <LineChart
```

Comment:
Is it possible to add prop to define the size of the chart metric?

Reply:
what do you mean by `size` here.. width?

Reply:
yes width